### ISSUE TITLE
Uploaded files not visible in edit mode

### ISSUE THREAD

**Issue by aostrow** [BEXIS2/Core#1975]
### What is going wrong?

After file upload to an File-entity the file are not visible in Edit-Mode anymore. 

step 1
![grafik](https://github.com/user-attachments/assets/d9a71cee-0e84-4566-ad19-09357814b18d)

step 2
![grafik](https://github.com/user-attachments/assets/a8777406-820d-4e43-9807-3e4d1e831b0a)

result: 
![grafik](https://github.com/user-attachments/assets/498e04ed-c4ba-4bd2-8b14-6f0ab707254b)

### How is it supposed to work?

Files and descriptions should be visible in the edit mode.
h

**Comment by geofranzi** [comment]
This is correct, as it is removed from the temp folder for import. Once imported it should not be in the list to import anymore. Imported files also do not have descriptions.
